### DISCUSSION TITLE
result = inference_detector()  result do not show bboxes

### DISCUSSION THREAD

**Discussion by zhaohai3424** [open-mmlab/mmdetection#9278] [Q&A] [answered]
I USE faster-rcnn_r50_fpn_1x_COCO.py  train my dataset successfully.

i use inference.py to test my picture,and many pictrues show the right detection,howerver some did not show any result, as below,pic1 is right.pic2 is wrong
![图片](https://user-images.githubusercontent.com/117725731/200536786-95a1cffb-123c-40a0-901a-dc5b714497d3.png)

![图片](https://user-images.githubusercontent.com/117725731/200536936-608c7e6f-fcf7-4bed-8af9-5074c7e91d18.png)
do you guys known the reason?i will appriciate it,thanks

**Comment by ZwwWayne** [comment, marked as answer]
This is because there are no boxes predicted after filtering by the score threshold. Therefore, you can try to make it smaller.
